symfony.lock

```diff
@@ -403,7 +352,7 @@
         "version": "1.0.x-dev"
     },
     "spryker-sdk/upgrader": {
-        "version": "dev-master"
+        "version": "dev-feature/stand-alone-plus-php74"
```

Comment:
I believe we can use dev-master. They merged php7.4 support into master couple of weeks ago